dev-tools/release/smoketest_rc.sh

```diff
@@ -0,0 +1,220 @@
+  echo "Please set JAVA_HOME to use JDK 8 to compile Geode for release"
+  exit 1
+fi
+if $JAVA -XshowSettings:properties -version 2>&1 | grep 'java.vm.vendor = Oracle' ; then
+  echo "Please set JAVA_HOME to use an Open JDK 8 such as from https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot to compile Geode for release"
+  exit 1
+else
+  $JAVA -XshowSettings:properties -version 2>&1 | grep 'java.vm.vendor = '
+fi
+
+set -x
+WORKSPACE=$PWD/smoketest-release-${VERSION}-workspace
+GEODE=$WORKSPACE/geode
+GEODE_EXAMPLES=$WORKSPACE/geode-examples
+GEODE_NATIVE=$WORKSPACE/geode-native
+GEODE_BENCHMARKS=$WORKSPACE/geode-benchmarks
+BREW_DIR=$WORKSPACE/homebrew-core
+SVN_DIR=$WORKSPACE/dist/dev/geode
+if which shasum >/dev/null; then
+  SHASUM="shasum -a 256"
+else
+  SHASUM=sha256sum
+fi
+set +x
+
+
+function failMsg1 {
+  echo "ERROR: script did NOT complete successfully.  Please try again."
+}
+trap failMsg1 ERR
+
+
+echo ""
+echo "============================================================"
+echo "Cleaning workspace directory..."
+echo "============================================================"
+set -x
+rm -rf $WORKSPACE
+mkdir -p $WORKSPACE
+cd $WORKSPACE
+set +x
+
+
+echo ""
+echo "============================================================"
+echo "Cloning repositories at the RC tag..."
+echo "============================================================"
+set -x
+mkdir src_clones
+cd src_clones
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode.git apache-geode-${VERSION}-src
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode-examples.git apache-geode-examples-${VERSION}-src
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode-native.git apache-geode-native-${VERSION}-src
+git clone --depth=1 --branch=rel/v${FULL_VERSION} git@github.com:apache/geode-benchmarks.git apache-geode-benchmarks-${VERSION}-src
+cd $WORKSPACE
+set +x
+
+
+echo ""
+echo "============================================================"
+echo "Downloading artifacts"
+echo "============================================================"
+
+DOWNLOAD_URL="https://dist.apache.org/repos/dist/dev/geode/$FULL_VERSION"
+set -x
+wget $DOWNLOAD_URL -m -np -nv
+set +x
+
+DOWNLOAD_DIR=$WORKSPACE/dist.apache.org/repos/dist/dev/geode/$FULL_VERSION
+echo ""
+echo "============================================================"
+echo "Untarring source releases"
+echo "============================================================"
+set -x
+mkdir -p src_releases
+cd src_releases
+ls $DOWNLOAD_DIR/*src.tgz | xargs -n1 -I {} bash -c "tar xzf {}"
+cd -
+set +x
+
+echo ""
+echo "============================================================"
+echo "Verifying Signatures"
+echo "============================================================"
+
+set -x
+cd $DOWNLOAD_DIR
+ls *.zip *.tar.gz *.tgz | xargs -n1 -I {} bash -c "shasum -c {}.sha*"
+ls *.zip *.tar.gz *.tgz | xargs -n1 -I {} bash -c "gpg --verify {}.asc"
```

Comment:
I got "Can't check signature: No public key" ... is there any setup work needed first, like downloading https://github.com/apache/geode/blob/develop/KEYS ?

Reply:
You probably do need to import the keys file, eg `gpg --import KEYS`, if you haven't already.